File: app/src/main/java/com/centaury/cataloguemovie/data/remote/ApiEndPoint.java
package com.centaury.cataloguemovie.data.remote;

import com.centaury.cataloguemovie.BuildConfig;

/**
 * Created by Centaury on 10/24/2019.
 */
public final class ApiEndPoint {

    public static final String ENDPOINT_DISCOVER_MOVIE = BuildConfig.BASE_URL + "discover/movie";
    public static final String ENDPOINT_GENRE_MOVIE = BuildConfig.BASE_URL + "genre/movie/list";

    public static final String ENDPOINT_DISCOVER_TVSHOW = BuildConfig.BASE_URL + "discover/tv";
    public static final String ENDPOINT_GENRE_TVSHOW = BuildConfig.BASE_URL + "genre/tv/list";


}

File: app/src/main/java/com/centaury/cataloguemovie/data/remote/ApiEndPoint.kt
package com.centaury.cataloguemovie.data.remote

import com.centaury.cataloguemovie.BuildConfig

/**
 * Created by Centaury on 10/24/2019.
 */
object ApiEndPoint {
    const val ENDPOINT_DISCOVER_MOVIE = BuildConfig.BASE_URL + "discover/movie"
    const val ENDPOINT_DETAIL_MOVIE = BuildConfig.BASE_URL + "movie/{movie_id}"
    const val ENDPOINT_GENRE_MOVIE = BuildConfig.BASE_URL + "genre/movie/list"
    const val ENDPOINT_DISCOVER_TV_SHOW = BuildConfig.BASE_URL + "discover/tv"
    const val ENDPOINT_DETAIL_TV_SHOW = BuildConfig.BASE_URL + "tv/{tv_id}"
    const val ENDPOINT_GENRE_TV_SHOW = BuildConfig.BASE_URL + "genre/tv/list"
}